pkg/apis/scheduling/types.go

```diff
@@ -278,6 +278,13 @@ type QueueStatus struct {
 	Reservation Reservation `json:"reservation,omitempty" protobuf:"bytes,6,opt,name=reservation"`
 }
 
+// CluterSpec represents the template of Cluster
+type Cluster struct {
+	Name string
+	Weight int32
```

Comment:
Is there some comments about field `Weight` and `Capacity`?

Reply:
The meaning of cluster Weight and Capacity is straightforward. It is similiar with Weight and Capacity in QueueSpec.

Reply:
> The meaning of cluster Weight and Capacity is straightforward. It is similiar with Weight and Capacity in QueueSpec.

OK, IC